slideview/build.gradle

```diff
@@ -22,6 +21,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    buildToolsVersion '27.0.3'
```

Comment:
This should be updated in the project-level build.gradle file and referenced as 
`buildToolsVersion rootProject.buildToolsVersion` so all modules will use the same version.